pygsti/report/plotly_plot_ex.py

```diff
@@ -118,33 +113,20 @@ def plot_ex(figure_or_data, show_link=True, link_text='Export to plot.ly',
     config['showLink'] = show_link
     config['linkText'] = link_text
 
-    #Add version-dependent kwargs to _plot_html call below
-    plotly_version = tuple(map(int, _plotly_version.split('.')))
-    if plotly_version < (3, 8, 0):  # "old" plotly with _plot_html function
-        from plotly.offline.offline import _plot_html
-
-        kwargs = {}
-        if plotly_version >= (3, 7, 0):  # then auto_play arg exists
-            kwargs['auto_play'] = False
-
-        #Note: removing width and height from layout above causes default values to
-        # be used (the '100%'s hardcoded below) which subsequently trigger adding a resize script.
-        plot_html, plotdivid, _, _ = _plot_html(
-            fig, config, validate, '100%', '100%',
-            global_requirejs=False,  # no need for global_requirejs here
-            **kwargs)  # since we now extract js and remake full script.
-    else:
```

Comment:
This change effectively kills support for plotly <3.8.0, so the minimum version requirement should be updated.